charts/studio/templates/configmap.yaml

```diff
@@ -1,9 +1,87 @@
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: studio-ca-certificates
+  name: studio
 data:
-  {{- if .Values.global.customCaCert }}
-  self_signed_ca.crt: |
-{{ .Values.global.customCaCert | indent 4}}
+  ALLOWED_HOSTS: "*"
+{{/*  {{- if .Values.studioBackend.ingress.enabled }}*/}}
+{{/*  {{- range $host := .Values.studioBackend.ingress.hosts }}*/}}
+{{/*    {{- range .paths }}*/}}
+{{/*  #*/}}
+{{/*  API_URL: "http{{ if $.Values.studioBackend.ingress.tls }}s{{ end }}://{{ $host.host }}{{ .path }}"*/}}
+{{/*    {{- end }}*/}}
+{{/*  {{- end }}*/}}
+{{/*  {{- else }}*/}}
+  API_URL: "http://studio-backend.{{ .Release.Namespace }}.svc.cluster.local:{{ .Values.studioBackend.service.port }}"
+{{/*  {{- end }}*/}}
+
+{{/*  UI_URL:*/}}
+{{/*  {{- if .Values.studioUi.ingress.enabled }}*/}}
+{{/*  {{- range $host := .Values.studioUi.ingress.hosts }}*/}}
+{{/*    {{- range .paths }}*/}}
+{{/*    UI_URL: "http{{ if $.Values.studioUi.ingress.tls }}s{{ end }}://{{ $host.host }}{{ .path }}"*/}}
+{{/*    {{- end }}*/}}
+{{/*  {{- end }}*/}}
+{{/*  {{- else }}*/}}
+  UI_URL: "http://studio-ui.{{ .Release.Namespace }}.svc.cluster.local:{{ .Values.studioUi.service.port }}"
```

Comment:
The workflow for enabled ingress is commented because of a bug https://github.com/iterative/helm-charts/issues/41

Reply:
same

Reply:
Same ^^